components/Op/OpBanner.js

```diff
@@ -50,7 +49,6 @@ const OpBanner = ({ op, children }) => {
           <h1>{op.name}</h1>
           <ItemIdLine item={op.offerOrg} path='orgs' />
           <h4>
-            {op.subtitle} <br />
             {op.location}
```

Comment:
I disagree that subtitle should not appear on the OpDetail page.
My reason is that the page is the target for shares on social media (not the OpCard). 
1. we want the shared presentation to look like the opCard as much as possible.  so the subtitle should be in the metadata and 'content' for the page for Search Engine results. 
2. Its likely volunteers will be directly straight to this page bypassing the OpCard.  The Subtitle is the Call to action for the volunteer - so needs to be prominent on the banner and definitely in the metadata.

If not present in the banner it should move to the top of the About panel and be the subheading there.

Reply:
1. agree that it should be in meta
2. happy to test it out but I'm suspecting it'll result in a crapload of double-ups where organisers explain themselves twice later on

Reply:
I've chucked it in for now though